db.sql

```diff
@@ -0,0 +1,88 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
Is this here just to keep a record? or do we use it to build the db?

Reply:
It provides the `uuid_generate_v4` function.